exchanges/bitfinex/bitfinex.go

```diff
@@ -118,6 +121,141 @@ func (b *Bitfinex) GetPlatformStatus() (int, error) {
 	return -1, fmt.Errorf("unexpected platform status value %d", response[0])
 }
 
+func baseMarginInfo(data []interface{}) ([]MarginInfoV2, error) {
+	var tempResp MarginInfoV2
+	tempData, ok := data[1].([]interface{})
+	if !ok {
+		return nil, fmt.Errorf("%w", errTypeAssert)
```

Comment:
This will just log an error to the console saying `type assertion failed` which is unhelpful. Could you please provide more details such as an exchange name from the calling function to help in the event there's an error?